--- poppler/glib/poppler-private.h.orig	2023-05-14 20:17:23.334710500 -0400
+++ poppler/glib/poppler-private.h	2023-05-14 20:18:37.916009400 -0400
@@ -150,7 +150,7 @@
 #define POPPLER_DEFINE_BOXED_TYPE(TypeName, type_name, copy_func, free_func)                                                                                                                                                                   \
     GType type_name##_get_type(void)                                                                                                                                                                                                           \
     {                                                                                                                                                                                                                                          \
-        static volatile gsize g_define_type_id__volatile = 0;                                                                                                                                                                                  \
+        static gsize g_define_type_id__volatile = 0;                                                                                                                                                                                  \
         if (g_once_init_enter(&g_define_type_id__volatile)) {                                                                                                                                                                                  \
             GType g_define_type_id = g_boxed_type_register_static(g_intern_static_string(#TypeName), (GBoxedCopyFunc)copy_func, (GBoxedFreeFunc)free_func);                                                                                    \
             g_once_init_leave(&g_define_type_id__volatile, g_define_type_id);                                                                                                                                                                  \
